README.md

```diff
@@ -0,0 +1,163 @@
+Portal URL need to be set. This can be done anywhere **before** the first call of the `TiAuth.initialize()`:
+
+    ```javascript
+    TiAuth.API_URL = 'https://api.travel-intelligence.com';
+    TiAuth.DASHBOARD_URL = 'https://portal.travel-intelligence.com';
+    ```
+
+2. **Use the `initialize` method** - to bootstrap your application:
+
+    ```javascript
+    TiAuth.initialize(function(token, user) {
+      //bootstrap your application here
+    });
+    ```
+
+  `token` is the authentication token needed to call Travel Intelligence services
+
+  `user` contains the user with its preferences
+
+## Logout
+
+`ti-auth.js` provides the ability to invalidate the current user session. It will automatically redirect the user to the login page.
+
+```javascript
+TiAuth.logout();
+```
+
+## Framework Integration
+`ti-auth.js` is build to be independent from any specific framework. This enables either a stand-alone use or the integration with common JavaScript frameworks.
+
+### Ember
+
+The initialization can be done in ember by creating a custom initializer.
+
+1. Import `ti-auth.js` in `ember-cli-build.js`:
+ 
+  ```javascript
+  app.import('bower_components/ti-auth.js/dist/ti-auth.min.js');
+  ```
+2. Create `app/initializers/ti-auth.js` with the following content:
+  ```javascript
+  import Ember from 'ember';
+
+  var TiAuthInitializer = {
+    name: 'waitForTiAuth',
+    initialize: function(registry, application) {
+
+      application.deferReadiness();
+
+      TiAuth.API_URL = 'https://staging-api.travel-intelligence.com';
+      TiAuth.DASHBOARD_URL = 'http://travel-intelligence.dev';
+
+      TiAuth.initialize(function(token, user) {
+        application.set('token', token);
+        application.advanceReadiness();
+      });
+    }
+  };
+
+  export
+  default TiAuthInitializer;
+  ```
+3. Import the initializer in the header of `app/app.js`:
+
+  ```javascript
+  import TiAuthInitializer from './initializers/ti-auth';
+  ```
+  .. and load it with the other modules:
+  ```javascript
+  loadInitializers(App, config.modulePrefix, TiAuthInitializer);
+  ```
+
+### Angular
+
+1. Remove the `ng-app` attribute from the page root element.
+
+2. Manually bootstrap the angular app in the call-back after initializing `ti-auth.js`.   
+
+  ```javascript
+  //configuration of the angular application
+  var app = angular.module('app', [ ... ]);
+
+  //initializing ti-auth.js
+  TiAuth.initialize(function(token) {
+    //bootstrap the angular app
+    angular.element(document).ready(function() {
+      yourApp.constant('token', token);
+      angular.bootstrap(document, ['app']);
+    });
+  });
+  ```
+
+## Developer Corner
+
+Just a few remarks for users intending to extend the `ti-auth.js` library.
+
+### Setup
+Clone the repository:
+
+    $ git clone git@github.com:travel-intelligence/ti-auth.js.git
+
+Install the required dependencies:
+
+    $ npm install
+
+### Tests
+
+    $ make test
+
+### Build
+
+    $ make
```

Comment:
Let’s add

---

or use

    $ make watch

to automatically watch for file changes and re-build the library.

### Distribute

To create a distributable file (that includes all modules and exposes the `TiAuth` object), run

    $ make dist